scadalts-ui/src/components/datasources/MetaDataSource/point.vue

```diff
@@ -535,6 +556,25 @@ export default {
 				return;
 			}
 		},
+		async validateScript() {
```

Comment:
It seems that these changes concern a different issue: https://github.com/SCADA-LTS/Scada-LTS/pull/2687